core/domain/user_services.py

```diff
@@ -701,8 +1151,15 @@ def parse_date_from_string(datetime_str):
 
 
 def get_user_impact_score(user_id):
-    """Returns user impact score associated with user_id"""
+    """Gets user impact score.
+
+    Args:
+        user_id: str. The user id.
 
+    Returns:
+        float. Representing user impact score associated with the given user_id.
+        Otherwise returns 0 if UserStatsModel does not exist for the given user_id.
```

Comment:
Strike "Otherwise" -- you haven't set up an if clause. Just say "Returns 0 if ..."